api/v1beta1/hyperconverged_types.go

```diff
@@ -480,6 +480,10 @@ type HyperConvergedStatus struct {
 	// DataImportCronTemplates is a list of the actual DataImportCronTemplates as HCO update in the SSP CR. The list
 	// contains both the common and the custom templates, including any modification done by HCO.
 	DataImportCronTemplates []DataImportCronTemplateStatus `json:"dataImportCronTemplates,omitempty"`
+
+	// SystemHealthStatus reflects the health of HCO and its secondary resources, based on the aggregated conditions.
+	// +optional
+	SystemHealthStatus string `json:"systemHealthStatus,omitempty"`
```

Comment:
I wonder why do we need this field. K8s is getting away from a single status fields. This is why we have conditions.

Reply:
The motivation was to have a way for users that don’t use Prometheus to check the operator's health. 

@sradco is this field necessary for these users? maybe checking the metrics endpoint (which will include the new metric), without accessing Prometheus UI could be enough?

Reply:
This is why we have the conditions.

Reply:
@fabiand - what do you think?

Reply:
Is there a condition which is providing this information (healthy or not). If there is a condition, then this is enough imo

Reply:
@fabiand  There is no single condition for this